kubernetes/doc/create-options.md

```diff
@@ -0,0 +1,122 @@
+# Kubernetes createOptions Extensions
+
+IoT Edge on Kubernetes supports several experimental features to allow a user to assign additional parameters to pods that represents IoT Edge Modules.
+
+## Experimental features
+
+All features described in this document are experimental. To enable them, the following environment variable need to be set for the edgeAgent (make note of the double underscores):
+
+| Environment Variable Name                   | value  |
+|---------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`             | `true` |
+| `ExperimentalFeatures__EnableK8SExtensions` | `true` |
+
+## Create Options
+
+We added CreateOptions for experimental features on Kubernetes. These options will be ignored until both `ExperimentalFeatures__Enabled` and `ExperimentalFeatures__EnableK8SExtensions` are not set to `true`.
```

Comment:
>ignored [](start = 86, length = 7)

Nit: Please change double negative to something like "options will take effect if both variable names are set to true"